### ISSUE TITLE
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe9 in position 1: invalid continuation byte

### ISSUE THREAD

**Issue by mgedmin** [ProgrammersOfVilnius/pov-server-page#51]
The nightly script emits this:

```
Traceback (most recent call last):
  File "/usr/lib/pov-server-page/du2webtreemap", line 180, in <module>
    main()
  File "/usr/lib/pov-server-page/du2webtreemap", line 172, in main
    data = "var tree = %s;" % format_json(json_data)
  File "/usr/lib/python2.7/json/__init__.py", line 244, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
UnicodeDecodeError: 'utf8' codec can't decode byte 0xe9 in position 1: invalid continuation byte
```

It seems to be a regression in version 2.0.0.

**Comment by mgedmin** [comment]
Ah, yes, in https://github.com/mgedmin/webtreemap-du/commit/1bcc14ab73fd8528297491f6672eb295ca19cf0f I made the du2webtreemap script Python-3-only, but then in #47 I forgot all about that and made pov-server-page-py2 use python2 in the shebang lines of all the scripts.

**Comment by mgedmin** [comment]
Actually I didn't make the script Python 3-only, I tried to make it bilingual but flubbed an if condition.

**Comment by mgedmin** [comment]
Fixed in 2.0.1.
